operational_analysis/toolkits/power_curve/functions.py

```diff
@@ -46,6 +47,8 @@ def pc_iec(x):
         for i in range(0, len(bins) - 1):
             idx = np.where((x >= bins[i]) & (x < bins[i + 1]))
             P[idx] = P_bin[i]
+        cutoff_idx = ((x < windspeed_start) | x > windspeed_end)
```

Comment:
Missing parentheses around `x > windspeed_end`. Also, minor point, but maybe it should be `x >= windspeed_end` since all other bins are defined as open on their upper bound? But I think it's fine the way it is for now.